tests/primer/packages_to_lint.json

```diff
@@ -0,0 +1,53 @@
+{
+  "scikit-learn": {
```

Comment:
Propose to remove as they have their `tests` in the main directory.

Reply:
f0be9abd235658eec8cf7e89268f908383d75616